src/contracts/storage/tick.rs

```diff
@@ -0,0 +1,483 @@
+extern crate alloc;
+
+use super::Pool;
+use math::types::{
+    fee_growth::FeeGrowth, liquidity::Liquidity, sqrt_price::calculate_sqrt_price,
+    sqrt_price::SqrtPrice,
+};
+use gstd::{Decode, Encode, TypeInfo};
+use decimal::*;
+use traceable_result::*;
+
+#[derive(Debug, Copy, Clone, Decode, Encode, PartialEq, TypeInfo)]
+#[codec(crate = gstd::codec)]
+#[scale_info(crate = gstd::scale_info)]
+pub struct Tick {
+    pub index: i32,
+    pub sign: bool,
+    pub liquidity_change: Liquidity,
+    pub liquidity_gross: Liquidity,
+    pub sqrt_price: SqrtPrice,
+    pub fee_growth_outside_x: FeeGrowth,
+    pub fee_growth_outside_y: FeeGrowth,
+    pub seconds_outside: u64,
+}
+
+pub const MAX_RESULT_SIZE: usize = 16 * 1024 * 8;
```

Comment:
Please remove constants such as `MAX_RESULT_SIZE`, `LIQUIDITY_TICK_LIMIT`, `POSITION_TICK_LIMIT`, and structs `PositionTick` and `LiquidityTick`.

These are variables and structures used for querying Invariant DEX and specific to the blockchain. Here, we will redefine them at the SDK creation stage